### ISSUE TITLE
Detokenization adds spaces to "o.s.frv."

### ISSUE THREAD

**Issue by HaukurPall** [mideind/Tokenizer#13]
I am testing the detokenization and noticed that the detokenization adds spaces between "o.s.frv." so it becomes  "o. s. frv."
```
tokenized = list(tokenizer.tokenize("o.s.frv.", normalize=False))
detokenized = tokenizer.detokenize(tokenized, normalize=False)
```

**Comment by HaukurPall** [comment]
The same seems to happen when dealing with URLs.
```
Expected :http://www.malfong.is
Actual   :http://www. malfong. is
```

**Comment by HaukurPall** [comment]
Also an issue in subsections: "1.1.1.1.1. Dráttarvélargerð" -> "1.1. 1.1. 1. Dráttarvélargerð"

**Comment by vthorsteinsson** [comment]
Many thanks for the feedback @HaukurPall. We've fixed this in version 2.0.3. https://github.com/mideind/Tokenizer/releases/tag/2.0.3.
